README.md

```diff
@@ -1,5 +1,8 @@
 The Polymer Renamer
 ===================
+
+[!Travis Build Status](https://travis-ci.org/PolymerLabs/PolymerRenamer.svg?branch=master)
```

Comment:
Thanks for the PR! For consistency with other Polymer projects, let's use this as the travis link:
[![Build Status](https://travis-ci.org/PolymerLabs/PolymerRenamer.svg?branch=master)](https://travis-ci.org/PolymerLabs/PolymerRenamer)